dashboard/app/controllers/aichat_controller.rb

```diff
@@ -160,6 +160,11 @@ def check_message_safety
     render(status: :ok, json: response_body)
   end
 
+  # GET /aichat/user_has_aichat_access
+  def user_has_aichat_access
+    render(status: :ok, json: current_user&.has_aichat_access?)
```

Comment:
Might be good to explicitly return a JSON object instead of a plain value? something like `{ hasAccess: true/false }`?

Alternatively, we could just return an `ok` response if they have access or a `403 Forbidden` if they don't

Reply:
Sounds good and updated!
I initially returned an object, but then saw that  `student_chat_history` returned an array instead of an {arrayName: array}. Would it be good to send an object to be more explicit about what the boolean represents?